File: MinMaxInArray/MinMaxInArray.cpp
#include<iostream>

void FindMinMax(int* arr, int size);

void FindMinMax(int* arr, int size){
    int max = 0;
    int min;

    for(int i=0;i<size;i++){
        if (max < arr[i]){
            max = arr[i];
        }
    }

    min = max;

    for(int i=0;i<size;i++){
        if (min > arr[i]){
            min = arr[i];
        }
    }

    std::cout << "Max: " << max << " Min: " << min << std::endl;
}

int main(){
    int* arr;
    int size;

    std::cout << "size of array: ";
    std::cin >> size;

    for(int i=0;i<size;i++){
        std::cin >> arr[i];
    }

    FindMinMax(arr, size);

    return 0;
}

File: MinMaxInArray/MinMaxInArray.py
#!/usr/bin/python3

def FindMax(arr):
    max = 0
    for i in arr:
        if i > max:
            max = i

    return max

def FindMin(arr, max):
    min = max
    for i in arr:
        if i < min:
            min = i
    return min

arr = [8, 7, 9, 10, 11, 54, 43, 34, 87]
max = FindMax(arr)
min = FindMin(arr, max)

print("Min: ", min, " Max: ", max)